tyk-pro/templates/NOTES.txt

```diff
@@ -1,45 +1,19 @@
-Run the following in bash to bootstrap the dashboard instance:
+- The Dashboard is bootstrapped automatically, to find Login details please run those commands inside your tyk namespace:
 
-1. Bootstrap the dashboard so we can get a username and password to login, this also generates access tokens for the controller to use
+For the URL: (kubectl get nodes --selector=kubernetes.io/role!=master -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}'), if you're using Minikube then: (minikube ip) would be sufficient"
+For the port: (kubectl get --namespace tyk -o jsonpath="{.spec.ports[0].nodePort}" services dashboard-svc-tyk-pro)"
+For username: (kubectl get secret --namespace tyk tyk-login-details -o jsonpath="{.data.TYK_USER}" | base64 --decode)
+For password: (kubectl get secret --namespace tyk tyk-login-details -o jsonpath="{.data.TYK_PASS}" | base64 --decode)
 
-{{- if .Values.dash.ingress.enabled }}
-export DASH_URL="{{ .Values.dash.hostName }}"
-export DASH_HTTPS="{{ if gt (len .Values.dash.ingress.tls) 0 }}1{{ end }}"
-{{- else }}
-export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services dashboard-svc-{{ include "tyk-pro.fullname" . }})
-export NODE_IP=$(kubectl get nodes --selector=kubernetes.io/role!=master -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}')
+> Please note, if you're using an ingress for the dashboard to find out your URL and port please see the steps below:
 
-If you're using minikube, run this instead:
-export NODE_IP=$(minikube ip)
+> URL can be accessed here: "{{ .Values.dash.hostName }}"
+> For the port run: (kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services dashboard-svc-{{ include "tyk-pro.fullname" . }})
```

Comment:
`NodePort` is not a useful thing with an ingress it won't be even exposed normally.

Reply:
`NodePort` is now removed.